### PR TITLE
Empty repeater field fix

### LINKED ISSUE #1297: Prevent Repeater field showing up in PDF if empty
Prevent empty Repeater fields showing up in the PDF.

### PR DESCRIPTION
**Pull request by jestonihpi**
## Description
This fixes the issue #1297

## Checklist:
- [x] I've tested the code.
- [x] My code is easy to read, follow, and understand
- [x] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation / docblocks.

### CODE REVIEW

**Review by jakejackson1 — CHANGES_REQUESTED** [review]
@jestonihpi can we add a unit test to verify this works as expected?

**Comment by jestonihpi** [comment]
Copy.

**Comment by jakejackson1** [comment]
@jestonihpi this works as expected if the Repeater field is not filled out at all. However, Gravity Forms does allow Repeaters to be Nested together and I'd like to see empty Nested Repeater fields removed from the PDF as well.

Here is an example of this occurring in Focus Gravity: [PDF.pdf](https://github.com/GravityPDF/gravity-pdf/files/9016310/PDF.pdf). The top-level repeater is filled out, but the two nested repeaters are not. 

Form used: [gravityforms-export-2022-06-30.json.txt](https://github.com/GravityPDF/gravity-pdf/files/9016309/gravityforms-export-2022-06-30.json.txt) (remove `.txt` extension from file before importing into Gravity Forms).

**Comment by jestonihpi** [comment]
Added unit test and updated the codebase @jakejackson1 . Thanks.

File: src/Helper/Fields/Field_Repeater.php [hunk 1]
```diff
@@ -187,7 +187,7 @@ public function get_repeater_html( $value, $field ) {
 				if ( $sub_field instanceof GF_Field_Repeater ) {
 
 					/* Only recursively output if a value exists */
-					if ( isset( $item[ $sub_field->id ] ) ) {
+					if ( ! empty( $item[ $sub_field->id ] ) ) {
```
Comment (jakejackson1) [inline src/Helper/Fields/Field_Repeater.php:190]: Would it make more sense to use `! $field->empty_deep( $item[ $sub_field->id ] )` here, to standardise it like the other empty checks?

File: src/Helper/Fields/Field_Repeater.php [hunk 2]
```diff
@@ -248,5 +248,13 @@ public function value() {
 		$this->cache( $this->get_value() );
 
 		return $this->cache();
+
+	}
+
+	public function maybe_show_section_title( $is_top_level, $value, $field ) {
```
Comment (jakejackson1) [inline src/Helper/Fields/Field_Repeater.php]: This probably doesn't need to be `public`.

**Pull request merged**